tests/unit/plugins/inventory/test_kubevirt.py

```diff
@@ -125,35 +118,31 @@
             "qosClass": "Burstable",
             "virtualMachineRevisionName": "revision-start-vm-12345",
             "volumeStatus": [
-                {
-                    "name": "cloudinit",
-                    "size": 1048576,
-                    "target": "vdb"
-                },
+                {"name": "cloudinit", "size": 1048576, "target": "vdb"},
                 {
                     "name": "containerdisk",
                     "target": "vda",
-                }
+                },
             ],
-        }
+        },
     },
 )
+COMPLETE_VMI_WITH_NETWORK_NAME = merge_dicts(
+    COMPLETE_VMI,
+    {"status": {"interfaces": [{"ipAddress": "10.10.10.10", "name": "test-network"}]}},
+)
 
 BASE_SERVICE = {
     "apiVersion": "v1",
     "kind": "Service",
-    "metadata": {
-        "name": "test-service"
-    },
-    "spec": {}
+    "metadata": {"name": "test-service"},
+    "spec": {},
 }
+LOADBALANCER_SERVICE_WITHOUT_AND_SELECTOR_PORTS = merge_dicts(
+    BASE_SERVICE, {"spec": {"type": "LoadBalancer"}}
+)
 BASE_LOADBALANCER_SERVICE = merge_dicts(
-    BASE_SERVICE,
-    {
-        "spec": {
-            "type": "LoadBalancer"
-        }
-    }
+    BASE_SERVICE, {"spec": {"type": "LoadBalancer"}}
```

Comment:
Those are the same?

Reply:
Nice catch. I've messed up a bit with the git rebase :sweat: